### ISSUE TITLE
Yaw control from the RC

### ISSUE THREAD

**Issue by alphaville** [QUB-ASL/bzzz#66]
Let us re-introduce the following line in [`main.src`](https://github.com/QUB-ASL/bzzz/blob/main/src/main.cpp#L94)

```c++
yawReferenceRad += radio.yawRateReferenceRadSec() * SAMPLING_TIME;
```

But,

- Make sure `radio.yawRateReferenceRadSec() ` is zero unless we move the left stick; equivalently,
- `yawReferenceRad` doesn't drift unnecessarily
- Make sure the magnitude of the values of `radio.yawRateReferenceRadSec()` are reasonable (experimentation will be needed)

**Comment by gunturiCM** [comment]
I have opened another _experimental_ branch to see if we can use the `yawRateReference` to control the `Yaw` and have the `yawreference = currentYaw`.

**Comment by gunturiCM** [comment]
This has been addressed and solved in #71. The code was tested and successfully merged.
